documents/dispute.md

```diff
@@ -0,0 +1,288 @@
+## Dispute
```

Comment:
Please check for all comments here as added in this PR - https://github.com/razorpay/razorpay-dot-net/pull/122/files.

Reply:
made changes

Reply:
I see we have missed comments.